zebra-rpc/src/methods.rs

```diff
@@ -613,7 +615,14 @@ where
                 match response {
                     zebra_state::ReadResponse::TransactionIdsForBlock(Some(tx_ids)) => {
                         let tx_ids = tx_ids.iter().map(|tx_id| tx_id.encode_hex()).collect();
-                        Ok(GetBlock::Object { tx: tx_ids })
+                        Ok(GetBlock::Object {
+                            hash: block.hash(),
+                            height: block
+                                .coinbase_height()
+                                .expect("all blocks should have a coinbase height"),
+                            size: SerializedBlock::from(block).as_ref().len(),
```

Comment:
Optional: could we use `zcash_serialized_size()` here (needs a trait import as well):

```suggestion
                            size: block.zcash_serialized_size(),
```